packages/shared/src/graphql/sources.ts

```diff
@@ -62,8 +62,6 @@ export interface Squad extends Source {
   memberPostingRole: SourceMemberRole;
   memberInviteRole: SourceMemberRole;
   referralUrl?: string;
-  banner?: string;
-  borderColor?: string;
```

Comment:
Kinda confusing to have this when it is not really part of the schema. Maybe we should extend this base interface if we want to modify its properties that are not found in the actual schema. Just my thoughts.